client/gutenberg/extensions/slideshow/create-swiper.js

```diff
@@ -1,7 +1,7 @@
 /**
  * External dependencies
  */
-import { merge } from 'lodash';
+import { mapValues, merge } from 'lodash';
```

Comment:
This actually reminded me of something. The block registration that was merged in Jetpack predates our move away from React, and to this day declares `lodash` as a dependency: https://github.com/Automattic/jetpack/blob/master/extensions/blocks/slideshow/slideshow.php#L27-L29. Since we've done away with the big one, it would be nice to have no dependencies at all. This would mean ditching `lodash` from `createSwiper` though. Do you think we should do that?

Reply:
Hmm, I kinda :heart: `lodash`, but I get the point that it makes sense to get rid of dependencies.

Easy enough to get rid of `mapValues`, but `merge` could be kind of a PITA to re-implement -- and it would be a bummer to lose this way of merging param objects with defaults :thinking:

Reply:
I can kind of go either way on this. I propose that after we land this PR, I'll open a new one in which there are 0 view-side dependencies, and we can see how ugly it makes the code and assess whether it's worth it?